README.md

```diff
@@ -1,3 +1,5 @@
+![Android](https://github.com/touchlab/KaMPKit/workflows/Android/badge.svg?branch=master) ![iOS](https://github.com/touchlab/KaMPKit/workflows/iOS/badge.svg?branch=master)
```

Comment:
These don't update for PR builds, so won't see status until merge. I might have a little bit of hubris updating the readme now instead of waiting for it to build once first.